scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -109,6 +111,19 @@ def main():
         output["context"]["clients"], int(arguments["--clients"]), "Client count doesn't match with JSON:", return_error
     )
 
+    # Test that the output of the TPC-H benchmark does not cause crashes in the compare_benchmarks.py script. Since this
+    # script expects the path to the TPC-H benchmark as sys.argv[1], we need to traverse up to the root of Hyrise.
```

Comment:
Explain why we don't call CompareBenchmarkScriptTest?

Reply:
Good point.
I think I overlooked this part. Don't really see a reason to keep it. Do you?

Reply:
Nope